docs/guide/protocols/relativehybridtopology.rst

```diff
@@ -1,4 +1,28 @@
 Relative Hybrid Topology Protocol
 =================================
 
-issue #743
\ No newline at end of file
+The relative free energy calculation approach calculates the difference in 
+free energy between two similar ligands. Depending on the ChemicalSystem 
+provided, the protocol either calculates the relative binding free energy 
+(RBFE), or the relative hydration free energy (RHFE). In a thermodynamic 
+cycle, one ligand is converted into the other ligand by alchemically 
+transforming the atoms that vary between the two ligands. The 
+transformation is carried out in both environments, meaning both in the 
+solvent (ΔG\ :sub:`solv`\) and in the binding site (ΔG\ :sub:`site`\) for RBFE calculations 
+and in the solvent (ΔG\ :sub:`solv`\) and vacuum (ΔG\ :sub:`vacuum`\) for RHFE calculations.
+
+.. _label: Thermodynamic cycle for the relative binding free energy protocol
+.. figure:: img/rbfe_thermocycle.png
+   :scale: 50%
+
+   Thermodynamic cycle for the relative binding free energy protocol.
+   
+The :class:`.RelativeHybridTopologyProtocol` uses a hybrid topology approach to represent the two
```

Comment:
Here we should make it clear that the Protocol DAG only runs a single leg of a thermodynamic cycle - i.e. you have to run two Protocol instances to get your ddG.